test/test_utils/get_test_data.jl

```diff
@@ -224,9 +224,17 @@ function build_c_sys5_re(; kwargs...)
             reserve_re[2],
             [collect(get_components(RenewableDispatch, c_sys5_re))[end]],
         )
+        add_service!(c_sys5_re, reserve_re[3], get_components(RenewableDispatch, c_sys5_re))
         for t in 1:2, (ix, serv) in enumerate(get_components(VariableReserve, c_sys5_re))
             add_forecast!(c_sys5_re, serv, Deterministic("get_requirement", Reserve_ts[t]))
         end
+        for t in 1:2, (ix, serv) in enumerate(get_components(ReserveDemandCurve, c_sys5_re))
```

Comment:
Nit: `ix` isn't used in this new code, so you don't need `enumerate`.  I see that you probably copy/pasted existing code that does the same thing.

Reply:
instead of enumerate you can use `eachvalue`

Reply:
or just `serv in get_components(ReserveDemandCurve, c_sys5_re)`

Reply:
Sorry, it was a copy-paste mistake